src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+
+public class Main {

```

Comment:
Please do not commit files/classes not required by the task

Reply:
For testing, you can use a ready-made test class: /src/test/java/core/basesyntax/WorkWithFileTest.java